sim/TEAMS.md

```diff
@@ -210,7 +210,7 @@ NICKNAME|SPECIES|ITEM|ABILITY|MOVES|NATURE|EVS|GENDER|IVS|SHINY|LEVEL|HAPPINESS,
 
 - `DYNAMAXLEVEL` is left blank for 10.
 
-- If `POKEBALL`, `HIDDENPOWERTYPE`, `GIGANTAMAX` and `DYNAMAXLEVEL`
+- If `POKEBALL`, `HIDDENPOWERTYPE`, `GIGANTAMAX`, `DYNAMAXLEVEL` and `TERATYPE`
```

Comment:
It should be explained that leaving TERATYPE blank defaults it to the Pokemon's first type

Reply:
@MathyFurret Thanks for the review. I've made a related note and this PR should be good to be merged.